src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/ReportRepository.kt

```diff
@@ -432,6 +434,28 @@ class OrderlyReportRepository(val isReviewer: Boolean,
         }
     }
 
+    override fun getReportRun(key: String): ReportRunLog
```

Comment:
You'll need tests for this in `ReportRunRepositoryTests`

Reply:
..and it should be easy enough to put in a test for the exception case that CodeCov is moaning about at the moment.